src/sections/Meshery/meshery.style.js

```diff
@@ -115,6 +118,15 @@ const MesheryWrapper = styled.section`
             }
         }
     }
+    @media only screen and (max-width: 479px) {
+        .info{
+            .description {
+                .desc-text{
+                    max-width:90%
+                }
+            }
+        }
+    }
```

Comment:
After the above change, we don't need this.

Reply:
OK, I will make these changes.